Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN \
       yarn \
   # The following packages are necessary to run headless-gl
```

Comment:
I'm not sure if this is a Dockerfile or bash comment. We might need the trailing `\` here as well, but let's try this first.